### DISCUSSION TITLE
elasticsuite 2.9.0 compatibility with magento 2.4.3

### DISCUSSION THREAD

**Discussion by vbidou** [Smile-SA/elasticsuite#2333] [Q&A]
Hello folks,
Does someone has the magic recipee in order to upgrade elasticsuite from 2.9 to 2.10 while you upgrade from magento 2.3.7 to 2.4.3 ?
I spent the whole day on it. I was able to upgrade magento to 2.4.3 successfully, and it seems that elasticsuite 2.9 was working.
Then I tried to upgrade elasticsuite from 2.9 to 2.10 (![image](https://user-images.githubusercontent.com/44623543/140622664-adc5cff3-bf0e-440e-a948-a51750e1c8df.png)), but the whole installation crashed miserably at the time of final database update "bin/magento setup:upgrade". The Elasticsearch application is crashing and it stops the setup:upgrade (![image](https://user-images.githubusercontent.com/44623543/140622699-55b7b311-0901-4e0d-af80-4c200e25d4d4.png))

If anyone has an idea how to properly execute this milestone, I will take any advise.
Thank you.

**Comment by romainruaud** [comment]
Hi @vbidou 

what do you have in configuration for Elasticsuite ? Do you have something in BDD and/or in the app/etc/env.php of your project ?

**Comment by vbidou** [comment]
Bonjour Romain, désolé pour mon retour tardif.
We have an installation of ElasticSuite which is configured to connect the Elasticsearch v7.6 which is on localhost.
In the last couple of days, we managed to identify an explosive mix for our server CPU/RAM capacities (Bot Crawlers + Elasticsearch hungriness + demanding setup:upgrade commands) which was causing the server to kill automatically the services. This is why we haven't had the possibility so far to upgrade elasticsuite to the most compatible version with magento 2.4, which is version 2.10. 
For the moment, we are running Magento 2.4.3 with Elasticsuite 2.9 and it seems to work. We will try to stabilize this magento version 2.4.3 first, before attempting to upgrade elasticsuite to 2.10. I will keep you posted here if we face any issue.
But for the moment, we consider that Elasticsuite 2.9 is compatible with Magento 2.4.3.
Best regards. François

**Comment by vbidou** [comment]
Hello,
Just to inform you that we successfully upgraded Elasticsuite to version 2.10 inside magento 2.4.3.
This issue we faced at the beginning was associated to physical server ressources (CPU and RAM) which weren't fully available.
My conclusion is that, during upgrade, it is better to ensure that server ressources are available at maximum and also block unwilling trafic like Bots crawllers.
I wanted to ensure that this post-mortem is documented. Thank you !
